packages/snap-preact/components/src/providers/theme.ts

```diff
@@ -74,7 +77,9 @@ export type ThemeResponsiveOverrides = {
 	desktop?: ThemeComponentsRestrictedOverrides;
 };
 
-export type ThemePartial = Omit<Theme, 'variables' | 'name'> & { variables?: ThemeVariablesPartial };
+export type responsiveKeys = 'default' | 'desktop' | 'tablet' | 'mobile' | undefined;
+
+export type ThemePartial = Omit<Theme, 'variables' | 'name'> & { variables?: ThemeVariablesPartial; activeBreakpoint?: responsiveKeys };
```

Comment:
I kinda think `activeBreakpoint` should be placed on the `Theme` type and not on `ThemePartial`...